content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -36,9 +36,9 @@ of **1000** services and **2000** sidecars with 70,000 mesh-wide requests per se
 After running the tests using Istio {{< istio_release_name >}}, we get the following results:
 
 - The Envoy proxy uses **0.5 vCPU** and **50 MB memory** per 1000 requests per second going through the proxy.
```

Comment:
Is this statement now not accurate when Telemetry V2 is in use?

Reply:
Yes, this number needs to be verified.

Reply:
@howardjohn says we just leave these numbers there for now, and he is planning to improve it in the future.